defaults/main.yml

```diff
@@ -1,54 +1,186 @@
+
+# State of the backup
+backuppc_srv_BackupsDisable: 0
+# Other possible values are 1: automatic backup disable, manual backup ok, 2: no backup at all 
+
+# Rsync default commands to backup and restore a host
+backuppc_srv_RsyncClientCmd: '$sshPath -x -q -l {{ backuppc_client_user }} $host sudo $rsyncPath $argList+'
+backuppc_srv_RsyncClientRestoreCmd: '$sshPath -q -x -l {{ backuppc_client_user }} $host sudo $rsyncPath $argList+'
+
+backuppc_srv_PingMaxMsec: 200
+# Transer Method for backups. Possible values: smb, rsync, rsyncd, tar, archive. Only rsync is tested in the role
+backuppc_srv_XferMethod: rsync
+backuppc_srv_XferLogLevel: 1
+
+# Name to use as the "from" name for email
+backuppc_srv_EMailFromUserName: '{{ backuppc_srv_unix_user }}'
+# Destination address to an administrative user who will receive a
+# nightly email with warnings and errors
+backuppc_srv_EMailAdminUserName: '{{ backuppc_srv_unix_user }}'
+# Destination domain name for email sent to users
+backuppc_srv_EMailUserDestDomain: ''
+
+#### CGI user interface configuration settings
+
+# Linux group of admin users through the CGI
+backuppc_srv_CgiAdminUserGroup: '{{ backuppc_srv_unix_user }}'
+# List of users who are also admin users through the CGI
+backuppc_srv_CgiAdminUsers: '{{ backuppc_srv_unix_user }}'
+
+# URL of the BackupPC_Admin CGI script.  Used for email messages.
+backuppc_srv_CgiURL: 'https://{{ backuppc_srv_server_name }}/backuppc/index.cgi'
+
+# BackupPC web interface language
+backuppc_srv_language: 'en'
+# Date display format for CGI interface 
+backuppc_srv_CgiDateFormatMMDD: 0 # International format
+
+## BackupPC web users configuration / htpasswd user entries
+backuppc_srv_web_users: []
+## List of web user accounts data for BackupPC 
+#
+# backuppc_srv_web_users:
+# - name: user1
+#   password: '{{ vault_secret }}'
+#   mail: user1@domain.org
+# - name: user2
+#   htpasswd_hash: 'user2:$apr1$70okuQfp$/PB1pKZ0YXi6cTGL/yHXs1' 
+#   mail: user2@domain.org
+#
+## The password can be defined explicitely, or only by its hash code. 
+## An htpasswd hash code for a user and a password can be obtaind running: 
+#
+# htpasswd -n user 
+#
+## htpasswd utility is installed in linux with apache2-utils package
+## You will be able to configure this users in `backuppc_server_web_main_user` 
+## and `backuppc_server_web_other_users` of backuppc_client role. 
+
+
+## Apache2 configuration data
+
+## Apache2 parameters
+# Flag, whether to configure apache2 or not 
+backuppc_srv_configure_apache: true
+# Flag to disable the default Apache2 VirtualHost (apache site)
+backuppc_srv_disable_apache_default_site: true
+
+# e-mail configured in Apache2 VirtualHosts
+backuppc_srv_apache_email: 'webmaster@{{ ansible_fqdn }}'
+
+# Configuration of SSLCertificateFile and SSLCertificateKeyFile for Certbot generated certificates
+
+# A flag as option automatically congfigure Certbot certificates or self-signed snakeoil pre-installed certs  
+backuppc_srv_apache_ssl_certbot: true 
```

Comment:
Maybe here the most tight default value would be `false`, since it's the most versatile case. Web access to a backup server is strangely exposed to the internet. 

With the current default value, a role execution error will be obtained in most cases.

Reply:
You're right, for the default value. For the internet exposure, at least [it's documented](https://github.com/UdelaRInterior/ansible-backuppc/pull/4/files?file-filters%5B%5D=.md#diff-04c6e90faac2675aa89e2176d2eec7d8R18). Such was the behaviour of backuppc debian package up to stretch. Just in buster debian changed Apache config. I just maintained our roles and stretch package behaviour. 
But ok, lets follow debian's trend and your advise. I code an optional custom restriction, that defaults to debian conf.

Reply:
Proposal also accepted, [see  v3.1.1](https://github.com/UdelaRInterior/ansible-backuppc/blob/e9b35873c85e9d5fc6c032722c360aaae4625856/defaults/main.yml#L174)

(I don't know why this conversation only says the code it's outdated and tells nothing about changes)

Please, resolve conversation if you agree.

And, with no more comments we can merge, in order to make official versions up to 3.1.1 (that we already use ourselves).